src/seedu/addressbook/data/person/Address.java

```diff
@@ -47,12 +38,4 @@ public boolean equals(Object other) {
                 && this.value.equals(((Address) other).value)); // state check
     }
 
-    @Override
-    public int hashCode() {
```

Comment:
Usually `equals(...)` and `hashCode()` are working together, so it is recommended that you abstract both of them or not at the same time.